source/level.js

```diff
@@ -201,44 +229,67 @@ export class Level extends Phaser.Scene{
         // Turn characterData into a json string and store it in a cookie
         let jsonString = JSON.stringify(this.characterData);
         document.cookie = "characterData=" + jsonString + ";" + expireString + ";path=/;";
-
     }
+
     addGold(addedGold){
         this.characterData.gold += addedGold;
         this.goldText.text = 'Gold: ' + this.characterData.gold;
     }
-    enemyKilled(){
+
+    enemyKilled(name){
         // Update kill quest score
-        if (this.characterData[this.background.name].enemiesKilled < this.killQuest) {
+        if (this.characterData[this.background.name].enemiesKilled[name] < this.killQuest) {
             this.characterData.totalEnemiesKilled++;
```

Comment:
Shouldn't this increment whether or not the quest is completed?

Reply:
Uhhhhhhh